### PR TITLE
New golang tutorial added by freecodecamp.org

### PR DESCRIPTION
**Pull request by subhadeepbanerjee2003**
## Changes proposed

- **New golang tutorial added by freecodecamp.org**

## Screenshots
Screenshot from 2023-08-20 22-12-28

## Note to reviewers

*I hope this small change will add value to the users and please give suggestions if further edit or modification needed*

### CODE REVIEW

File: database/languages/golang.json [hunk 1]
```diff
@@ -27,4 +27,12 @@
     "category": "languages",
     "subcategory": "golang"
   }
+  ,
+  {
+    "name": "Go Programming – Golang Course with Bonus Projects",
+    "description": "Learn the Go programming language in this full course for beginners. You'll practice writing performant, idiomatic Go with these hands-on lessons and challenges.",
```
Comment (CBID2) [inline database/languages/golang.json:33]:
```suggestion
    "description": "In this beginner-level Golang course, you'll practice writing performant, idiomatic Go with these hands-on lessons and challenges.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @subhadeepbanerjee2003! :) Other than the suggestions I presented, your PR is off to a great start.

**Comment by subhadeepbanerjee2003** [comment]
@CBID2 i have done you can check as per your time thanks :)

**Review by CBID2 — APPROVED** [review]
LGTM

**Comment by subhadeepbanerjee2003** [comment]
@Anmol-Baranwal @rupali-codes please check

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

> Btw 9hr Tutorial :)

> I would suggest codecademy to learn Go

**Comment by Anmol-Baranwal** [comment]
@subhadeepbanerjee2003 
I am merging it for now. Just keep the structure consistent from next time.

image

**Comment by subhadeepbanerjee2003** [comment]
Ok thanks and I will surely keep the structure in mind

**Pull request merged**